setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = pychart-miningape
+version = 0.0.0
+author = Kyle Craig Johnson, Daniel Lyakhovich, Jacob Paw Moeler, Arda Onal, Eren Polat, Robert Spralja
+author_email = miningape@gmail.com
+description = test
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/miningape/pychart
+project_urls =
+    Bug Tracker = https://github.com/miningape/pychart/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.9
+
+[options.packages.find]
+where = src
```

Comment:
Should it maybe be src/pychart? not sure how it works if we want to hide certain files from the package

Reply:
If you want internal modules append `_` I don't think it actually hides anything but it is private by convention.

Reply:
> Should it maybe be src/pychart? not sure how it works if we want to hide certain files from the package

It cannot be if it is `src/pychart` then you can't import pychart but only its submodules

Reply:
Apparently we can use __init__.py to control this explicitely, rather than appending `_` and hoping the user doesn't use it... You can take a look here: https://stackoverflow.com/questions/69920761/how-to-hide-internal-modules-in-a-packages-namespace